### PR TITLE
Some hog rewords and tweaks

### PR DESCRIPTION
**Pull request by pamelafox**
This PR reflects the changes in the other repo:
https://github.com/Cal-CS-61A-Staff/berkeley-cs61a/pull/3824/files

Renaming two of the rules for inclusivity reasons, clarifying a comment, DRYing up some code.

Also a few tweaks to code that node complained about.

I have not been able to test this locally yet, can you clarify what the steps are? I've tried npm run build and npm run dev, but I don't see a rendered hog on localhost. Am I missing a step? (A readme would be great!)

### CODE REVIEW

File: hog/package.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
     "nodemon": "^2.0.4",
     "react": "^16.12.0",
     "react-dom": "^16.12.0",
-    "react-scripts": "3.3.0"
+    "react-scripts": "^4.0.1"
```
Comment (pamelafox) [inline hog/package.json:15]: Upgraded due to an error in running the scripts. This upgrade did fix the error.

**Comment by rahularya50** [comment]
I'll add a README, sure. To develop, run `yarn dev` and it'll start everything up. You need to start both the webpack and flask servers for it to work. We use `yarn` for dependencies, so I think you may need to update the yarn.lock as well if you bump a dependency - I'll do that for this PR.

**Review by rahularya50 — APPROVED** [review]

**Comment by rahularya50** [comment]
The PR build looked good to me, so I merged.

**Comment by pamelafox** [comment]
great thanks Rahul!

**Pull request merged**